pkg/launcher/uninstall/uninstall.go

```diff
@@ -0,0 +1,54 @@
+package uninstall
+
+import (
+	"context"
+	"errors"
+	"log/slog"
+	"os"
+
+	"github.com/kolide/launcher/ee/agent"
+	"github.com/kolide/launcher/ee/agent/types"
+)
+
+func Uninstall(ctx context.Context, k types.Knapsack) {
+	uninstallNoExit(ctx, k)
+
+	if err := disableAutoStart(ctx); err != nil {
```

Comment:
why isn't this in `uninstallNoExit`?

Reply:
Hrm... This raises some interesting questions.

I think you're saying that testing this locally, triggers the uninstall actions, which might uninstall/disable your prod launcher. 

Does it make sense to allow passing the identifier in, or some other test mode that uses different files?

Reply:
we could do that or create something like an `autoStartDisabler` interface that we pass in, the test could at least verify that a func like `disableAutoStart` was called

Reply:
Or maybe have it take a boolean that CI passes differently. Or use the compiled constant trick.

But it feels weird to have it in two methods.

Reply:
Combined the functions